docs/source/handling_advanced_stypes/handle_text.rst

```diff
@@ -110,6 +110,9 @@ Fusing Text Embeddings into Tabular Learning
 to encode pre-computed embeddings. This encoder applies linear function over the
 pre-computed embeddings, which can easily handle :obj:`~torch_frame.stype.text_embedded`.
 
+Internally, :class:`~torch_frame.stype.text_embedded` is stored using the same data structure as :class:`~torch_frame.stype.embedding`.
+So we only need to specify the encoder for parent :obj:`~torch_frame.stype`, i.e. :class:`~torch_frame.stype.embedding`, in the :obj:`stype_encoder_dict`.
```

Comment:
"is stored using the same data structure as" is a bit vague to me.

Can we say something like -- is grouped in the parent stype "stype.embedding" withiin `TensorFrame`,